websites/T/The Odin Project/presence.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable unicorn/prefer-dom-node-text-content */
```

Comment:
Why have you removed this rule? For the most part, `textContent` is preferred over the use of `innerText` on the repo